client/reader/sidebar/index.jsx

```diff
@@ -172,18 +172,21 @@ export class ReaderSidebar extends Component {
 				<QueryReaderTeams />
 				<QueryReaderOrganizations />
 
+				<li className="sidebar-header">
+					<h3>{ translate( 'Reader' ) }</h3>
+					<p>{ translate( 'Keep up with your interests.' ) }</p>
+				</li>
```

Comment:
Added this for:

> Add a title and description at the top of the sidebar.

As per designs the description should be "Stay current with blogs you've subscribed to." but it was overflowing to 2 lines as the width of the sidebar available on designs is different than the actual width so I have used a shorter text for description.